src/routes.js

```diff
@@ -207,6 +208,14 @@ export class Router extends Component {
                   exact
                 />
 
+                <ProtectedRoute
+                  path="/gigs/:id"
+                  component={GigsView}
+                  isAuth={isAuth}
+                  onAuthCheck={onAuthCheck}
+                  exact
+                />
+
```

Comment:
can we add a unit test in `routes.test.js`

Reply:
done